robot_calibration/src/led_finder.cpp

```diff
@@ -21,6 +21,8 @@
 #include <robot_calibration/capture/led_finder.h>
 #include <sensor_msgs/point_cloud2_iterator.h>
 #include <sensor_msgs/image_encodings.h>
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/types_c.h>
```

Comment:
these includes should not be needed, given that we haven't added anything related to them?